### PR TITLE
Fix: Correct the typo in method name

### PR DESCRIPTION
**Pull request by Sauter001**
- The typo of the name of the function getHightLightDaysMap was corrected. (getHightLightDaysMap -> getHighLightDaysMap )

### CODE REVIEW

**Comment by martijnrusschen** [comment]
Are you expecting to follow up on this pull request?

File: .github/pull_request_template.md [hunk 1]
```diff
@@ -8,21 +8,64 @@ assignees: ""
 
 ## Description
 
-**Linked issue**: #(issue number) 
+<<<<<<< HEAD
```
Comment (martijnrusschen) [inline .github/pull_request_template.md:11]: This is unrelated

**Comment by mirus-ua** [comment]
Hello

[This one](https://github.com/Hacker0x01/react-datepicker/issues/4700) will be major, so you can update your PR to include it in the release
